routing/routes.go

```diff
@@ -497,6 +497,90 @@ func NewRoutes() (r *Routes) {
 			handlers.WorkloadMetrics,
 			true,
 		},
+		// swagger:route GET /namespaces/{namespace}/services/{service}/dashboard services serviceDashboard
+		// ---
+		// Endpoint to fetch dashboard to be displayed, related to a single service
+		//
+		//     Produces:
+		//     - application/json
+		//
+		//     Schemes: http, https
+		//
+		// responses:
+		//      400: badRequestError
+		//      503: serviceUnavailableError
+		//      200: dashboardResponse
+		//
+		{
+			"ServiceDashboard",
+			"GET",
+			"/api/namespaces/{namespace}/services/{service}/dashboard",
+			handlers.ServiceDashboard,
+			true,
+		},
+		// swagger:route GET /namespaces/{namespace}/apps/{app}/dashboard apps appDashboard
+		// ---
+		// Endpoint to fetch dashboard to be displayed, related to a single app
+		//
+		//     Produces:
+		//     - application/json
+		//
+		//     Schemes: http, https
+		//
+		// responses:
+		//      400: badRequestError
+		//      503: serviceUnavailableError
+		//      200: dashboardResponse
+		//
+		{
+			"AppDashboard",
+			"GET",
+			"/api/namespaces/{namespace}/apps/{app}/dashboard",
+			handlers.AppDashboard,
+			true,
+		},
+		// swagger:route GET /namespaces/{namespace}/workloads/{workload}/dashboard workloads workloadDashboard
+		// ---
+		// Endpoint to fetch dashboard to be displayed, related to a single workload
+		//
+		//     Produces:
+		//     - application/json
+		//
+		//     Schemes: http, https
+		//
+		// responses:
+		//      400: badRequestError
+		//      503: serviceUnavailableError
+		//      200: dashboardResponse
+		//
+		{
+			"WorkloadDashboard",
+			"GET",
+			"/api/namespaces/{namespace}/workloads/{workload}/dashboard",
+			handlers.WorkloadDashboard,
+			true,
+		},
+		// swagger:route GET /namespaces/{namespace}/apps/{app}/cstmetrics/{template} apps customDashboard
```

Comment:
cstmetrics -> custommetrics
I prefer to have descriptive names, mixing long names with acronyms creates a own acronyms game hard to memorize.
I don't mind to type but have clear the endpoint.